WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -171,20 +177,22 @@ private extension BlazeCampaignDashboardView {
         Button {
             createCampaignTapped?(nil)
         } label: {
-            Text(Localization.promote)
-                .foregroundColor(.init(uiColor: .accent))
+            Text(Localization.createCampaign)
+                .fontWeight(.bold)
                 .bodyStyle()
         }
+        .buttonStyle(SecondaryButtonStyle())
     }
 
     var showAllCampaignsButton: some View {
         Button {
             viewModel.didSelectCampaignList()
             showAllCampaignsTapped?()
         } label: {
-            HStack {
-                Text(Localization.showAllCampaigns)
+            HStack(spacing: 0) {
+                Text(viewModel.viewCampaignsButtonLabel)
                     .fontWeight(.regular)
+                    .foregroundColor(.accentColor)
```

Comment:
Nit: we should migrate to `foregroundStyle` instead because `foregroundColor` is deprecated in a future SwiftUI version.

Reply:
thanks for the reminder https://github.com/woocommerce/woocommerce-ios/pull/12593/commits/cee7c972b1aae8fa8bda93192b2f3e2f832167a2